aml-development/frontend
# feat(notifications): full notifications page

AMLOS-699

**Description**
The user should be able to view all notifications on a separate page after clicking the See More button on their notifications dropdown.

**Acceptance Criteria**
The user can click the see more button from the notifications dropdown to take them to a page to view and interact with all their notifications.

**How to Test**
Pull this branch
Click the notifications icon from the header and click 'See More'
Ensure the notifications are displayed and the user can interact with them like in the popup (ie delete, mark read/unread, etc)
